controller/scheduler/fleet.py

```diff
@@ -325,7 +325,7 @@ def attach(self, name):
     {"section": "Service", "name": "ExecStartPre", "value": '''/bin/sh -c "IMAGE=$(etcdctl get /deis/registry/host 2>&1):$(etcdctl get /deis/registry/port 2>&1)/{image}; docker pull $IMAGE"'''},  # noqa
     {"section": "Service", "name": "ExecStartPre", "value": '''/bin/sh -c "docker inspect {name} >/dev/null 2>&1 && docker rm -f {name} || true"'''},  # noqa
     {"section": "Service", "name": "ExecStart", "value": '''/bin/sh -c "IMAGE=$(etcdctl get /deis/registry/host 2>&1):$(etcdctl get /deis/registry/port 2>&1)/{image}; port=$(docker inspect -f '{{{{range $k, $v := .ContainerConfig.ExposedPorts }}}}{{{{$k}}}}{{{{end}}}}' $IMAGE | cut -d/ -f1) ; docker run --name {name} {memory} {cpu} {hostname} -P -e PORT=$port $IMAGE {command}"'''},  # noqa
-    {"section": "Service", "name": "ExecStop", "value": '''/usr/bin/docker rm -f {name}'''},
+    {"section": "Service", "name": "ExecStop", "value": '''/usr/bin/docker stop {name}; sleep 10; /usr/bin/docker rm -f {name}'''},  # noqa
```

Comment:
Rather than requiring every app unit to take 10s to shut down, I think we can sim;ly create two `ExecStop` entries - one to stop the container, and another to remove it. If necessary, we can set `TimeoutStopSec` as well.